packages/flutter_localizations/test/date_picker_test.dart

```diff
@@ -223,6 +223,59 @@ void main() {
 
     await tester.tap(find.text('ANNULER'));
   });
+
+  group('screen configurations', () {
+
+    const Size kPixel1Portrait = Size(1070, 1770);
```

Comment:
As before, maybe there's something to say about the sizes

Reply:
Done.